ViewGenerator/build/ViewGenerator.props

```diff
@@ -6,8 +6,12 @@
     <None Include="ts2lang.json">
       <Visible>True</Visible>
     </None>
+    <Content Include="**\*.ts" />
```

Comment:
hum... this will include these files twice, because they should be included by the developer

Reply:
I'll remove this. I was thinking that by default Visual Studio put ts and tsx files to compile with TypeScriptCompile target, and this would override that, but since we have TypeScriptCompile blocked, this is useless